p2p/protocol/identify/id.go

```diff
@@ -72,6 +80,31 @@ func NewIDService(ctx context.Context, h host.Host) *IDService {
 		currid:        make(map[network.Conn]chan struct{}),
 		observedAddrs: NewObservedAddrSet(ctx),
 	}
+
+	// handle local protocol handler updates, and push deltas to peers.
+	s.subscriptions.evtLocalProtocolsUpdated = make(chan event.EvtLocalProtocolsUpdated, 128)
+	cancelFunc, err := h.EventBus().Subscribe(s)
+	if err != nil {
+		log.Warningf("identify service not subscribed to local protocol handlers updates; err: %s", err)
+	} else {
+		go func() {
+			var evt event.EvtLocalProtocolsUpdated
+			for {
+				select {
+				case evt = <-s.subscriptions.evtLocalProtocolsUpdated:
```

Comment:
Cleaner to just do := in the select case, instead of declaring the variable ahead of time